### ISSUE TITLE
Create references on content of HIT upon search

### ISSUE THREAD

**Issue by bidoubiwa** [meilisearch/documentation#748]
In a search response you typically have the following

```js
{
  hits: []
  query: ""
}
```

Inside `hits` you have multiple hit. These hit not only have the expected content of the document (`id`, `title` in the case of movies for example) but also additional fields depending on your search parameters: 

For example `_matchesInfo` or `_formatted`.
```js
{
  "id": "50393",
  "title": "Kung Fu Panda Holiday",
  "_formatted": {
    "overview": "The Winter Feast is Po's favorite holiday. Every year he and his father hang decorations, cook together, and serve noodle soup to the villagers. But this year <em>Shifu</em> informs Po that as Dragon Warrior, it is his duty to host the formal Winter Feast at the Jade Palace. Po is caught between his obligations as the Dragon Warrior and his family traditions: between <em>Shifu</em> and Mr. Ping.",
  },
  "_matchesInfo": {
    "overview": [
      {
        "start": 159,
        "length": 5
      },
      {
        "start": 361,
        "length": 5
      }
    ]
  }
}
 
```

I think the content of `hit` should be described atleast in the API references.

**Comment by maryamsulemani97** [comment]
closing as the  content of `hit` is documented in the [Search API reference](https://docs.meilisearch.com/reference/api/search.html)

**Comment by dichotommy** [comment]
Closed in favor of newer issue #1626

Will still be done! Never fear 🦁
